docs/galaxy/docsite/rst/404.rst

```diff
@@ -0,0 +1,12 @@
+:orphan:
+
+*****
+Oops!
+*****
+
+The version of the Galaxy documentation you were looking at doesn't contain that page.
+
+.. image:: images/cow.png
+   :alt: Cowsay 404
+
+Use the back button to return to the version you were browsing, or use the navigation at left to explore our latest release. Once you're on a non-404 page, you can use the version-changer to select a version.
```

Comment:
Do we need to update the docsite repo too, to make sure the `ansible-galaxy` directory does the right thing for 404s?